content/5_Plat/Bin_Jump.mdx

```diff
@@ -353,6 +353,86 @@ int main() {
 
 <IncompleteSection />
 
+### Solution (Company Queries II)
+
+First, let $\texttt{st}, \texttt{en}, and \texttt{up}$ be the time-in, time-out, and ancestor table for all nodes in the tree. 
+
+These can be filled with a DFS traversal. $\texttt{up}$ can be generated because depth first allows the ancestors to be filled first before traversing the current node.
```

Comment:
@dolphingarlic do you think that this problems needs an explanation? (lca and its implementations are already covered in resources). I think it suffices to say something alone the lines of "use your favorite lca impl" - or in this case binary lifting.

Reply:
obviously it wouldn't hurt to have though